crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -1,14 +1,55 @@
+//! The modular rendering abstraction responsible for queuing, preparing, sorting and drawing
+//! entities as part of separate render phases.
+//!
+//! In Bevy each view (camera, or shadow-casting light, etc.) has one or multiple [`RenderPhase`]s
+//! (e.g. opaque, transparent, shadow, etc).
+//! They are used to queue entities for rendering.
+//! Multiple phases might be required due to different sorting/batching behaviours
+//! (e.g. opaque: front to back, transparent: back to front) or because one phase depends on
+//! the rendered texture of the previous phase (e.g. for screen-space reflections).
+//!
+//! To draw an entity, a corresponding [`PhaseItem`] has to be added to one or multiple of these
+//! render phases for each view that it is visible in.
+//! This must be done in the [`RenderStage::Queue`](crate::RenderStage::Queue).
+//! After that the render phase sorts them in the
+//! [`RenderStage::PhaseSort`](crate::RenderStage::PhaseSort).
+//! Finally the items are rendered using a single [`TrackedRenderPass`], during the
```

Comment:
I do not think that this is true.
A draw function and the render commands take a `TrackedRenderPass` as an argument.

Theoretically, you could iterate over the items of the render phase manually and draw them with custom logic, but is that an actual use case?

Reply:
Hum... I just looked at the `bevy_hanabi` code as I thought I had avoided the `TrackedRenderPass` because it's just overhead when you don't need it, but indeed it's a parameter so you can't avoid it. Nevermind then you can ignore that comment. Thanks!